hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/config/GsonConfig.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.config
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.google.gson.JsonDeserializationContext
+import com.google.gson.JsonDeserializer
+import com.google.gson.JsonElement
+import com.google.gson.JsonPrimitive
+import com.google.gson.JsonSerializationContext
+import com.google.gson.JsonSerializer
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.lang.reflect.Type
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+
+@Configuration
+class GsonConfig() {
```

Comment:
I actually removed this file from indexer and switched to object mapper instead.  Jackson is the default with spring boot so made more sense to just use that out of the box